UsefulProteomicsDatabases/ProteinDbLoader.cs

```diff
@@ -309,6 +315,11 @@ public static List<Protein> LoadProteinXML<T>(string proteinDbLocation, bool onT
                                                     decoyBigPeptideTypes[oneBasedBeginPositions.Count - i - 1] = peptideTypes[i];
                                                 }
 
+                                                foreach (DisulfideBond disulfideBond in disulfideBonds)
+                                                {
+                                                    decoy_disulfides.Add(new DisulfideBond(reversed_sequence.Length - disulfideBond.OneBasedBeginPosition + 1, reversed_sequence.Length - disulfideBond.OneBasedEndPosition + 1, disulfideBond.Description));
```

Comment:
I thought for decoys the disulfides should be at the same cysteine positions in the reverse sequence